packages/@lightningjs/ui-components/src/docs/Introduction.stories.mdx

```diff
@@ -32,6 +32,6 @@ In **The Canvas** we can see how the component stands in isolation as well as ad
 
 With **The Docs** panel, we can read through the use case for the component, see how it is implemented in code, view some examples, and see a table of its configurable properties and styles.
 
-For information on leveraging `lightning-ui-components` in your application, check out the [README.md](../?path=/story/docs-read-me).
+For information on leveraging `lightning-ui-components` in your application, check out the [README.md](../?path=/story/docs-read-me--page).
```

Comment:
Just curious, Is the path of README.md you have updated? Because it is not in the docs section?
Or Did I get it wrong?

Reply:
Not sure when it changed, but it seams like it adds page to all the urls now.